operator/cmd/mesh/install.go

```diff
@@ -247,7 +247,13 @@ func InstallManifests(iop *v1alpha12.IstioOperator, force bool, dryRun bool, kub
 		return iop, fmt.Errorf("errors occurred during operation: %v", err)
 	}
 	if status.Status != v1alpha1.InstallStatus_HEALTHY {
-		return iop, fmt.Errorf("errors occurred during operation")
+		additionalInfo := ""
```

Comment:
This seems unrelated to this PR

Reply:
Yes, i used it to know why the install failed